metaflow/plugins/cards/card_decorator.py

```diff
@@ -66,16 +65,6 @@ def _load_card_package(self):
             file_path, arcname = path_tuple
             yield (file_path, os.path.join("metaflow", "plugins", "cards", arcname))
 
-        external_card_pth_generator = _get_external_card_package_paths()
-        if external_card_pth_generator is None:
-            return
-        for module_pth, parent_arcname in external_card_pth_generator:
-            # `_get_card_package_paths` is a generator which yields
-            # path to the module and its relative arcname in the metaflow-extensions package.
-            for file_pth, rel_path in self._walk(module_pth, prefix_root=True):
-                arcname = os.path.join(parent_arcname, rel_path)
-                yield (file_pth, arcname)
-
     def _walk(self, root, filter_extensions=[], prefix_root=False):
```

Comment:
You should be able to remove all of `add_to_package`, _load_card_package` and `_walk`.